app/preprints/-components/submit/metadata/template.hbs

```diff
@@ -83,7 +83,7 @@
                             @provider={{@manager.provider}}
                             @metadataChangeset={{this.metadataFormChangeset}} 
                             @doesAutosave={{true}}
-                            @hasSubjects={{action this.hasSubjects}}
+                            @hasSubjects={{perform this.hasSubjects}}
```

Comment:
We won't need to use `perform` here if we change `this.hasSubjects` to an action, per the previous comment